packages/unmock-node/src/backend/index.ts

```diff
@@ -75,11 +75,16 @@ export default class NodeBackend implements IBackend {
   } = {};
   private origOnSocket?: (socket: net.Socket) => void;
   private mitm: any;
+  private stateStore: any;
 
   public constructor(config?: INodeBackendOptions) {
     this.config = { ...nodeBackendDefaultOptions, ...config };
   }
 
+  get states() {
+    return this.stateStore;
```

Comment:
I'd perhaps expect the initialize method to return this rather than have users access it on fly (with states being possibly undefined)?

Reply:
Ok I see it's needed for modifying the state, personally I still it's a bit too tight coupling to give users access to something that's also directly used internally, I would have maybe expected the exposed thing to be something like `export const states = new DSL(backend.stateStore)` but it's not a big deal.

Reply:
Sure! What would the wrapper do though?

Reply:
Wrap calls to internal functions like `updateState` with DSL and proxy commands, but I guess that's what the state store is now doing? Forgot what that looks like...

Reply:
Checked out `stateStoreFactory`, that's pretty much doing what I was thinking to be the `new DSL` so nevermind!

Reply:
Returning it from call to `initialize()` is a good idea! Otherwise `states` is undefined.